fab/inventory/staging-va

```diff
@@ -43,5 +43,7 @@ hqdb0-staging.internal-va.commcarehq.org
 hqdb0-staging.internal-va.commcarehq.org
 
 [riakcs]
+hqdjango4-staging.internal.commcarehq.org
```

Comment:
Actually, I'm not sure it will matter either way. The credentials will be updated on the next localsettings deployment, but the old ones will continue to work, so it shouldn't break anything.